src/screens/UserPortal/Donate/Donate.spec.tsx

```diff
@@ -188,6 +194,237 @@ describe('Testing Donate Screen [User Portal]', () => {
     expect(screen.getByPlaceholderText('Amount')).toBeInTheDocument();
   });
 
+  test('Donation amount input should update state', async () => {
+    render(
+      <MockedProvider addTypename={false}>
+        <BrowserRouter>
+          <Provider store={store}>
+            <I18nextProvider i18n={i18nForTest}>
+              <Donate />
+            </I18nextProvider>
+          </Provider>
+        </BrowserRouter>
+      </MockedProvider>,
+    );
+
+    const amountInput = screen.getByPlaceholderText('Amount');
+    fireEvent.change(amountInput, { target: { value: '100' } });
+    expect(amountInput).toHaveValue('100');
```

Comment:
we are not checking the state here (as the test description says)

Reply:
Sorry, I see why it's tested that way!

Reply:
Thanks for your feedback!